packages/api/src/auth/authDecoders/magicLink.ts

```diff
@@ -0,0 +1,32 @@
+import { AuthenticationError } from 'apollo-server-lambda'
```

Comment:
I would make magicLink, custom, and firebase share the same decoder.

Reply:
Right, essentially a `token` "decoder".